src/actinia_core/core/common/api_logger.py

```diff
@@ -110,7 +110,9 @@ def add_entry(self, user_id, http_request):
 
         """
         api_info = {
-            "endpoint": http_request.endpoint,
+            # For depricated location endpoints remove "_locations" from
+            # endpoint class name
+            "endpoint": http_request.endpoint.replace("_locations", ""),
```

Comment:
Why is this necessary?

Reply:
It is needed because I can't add two urls for one endpoint. The endpoint is inside the actinia response and is used for the job jesumption.

Reply:
Ah I see. Would it be possible to use the variable `projects_url_part`? (Probably not because of circular reference)

Reply:
As discussed we leave it like this to be able to find and remove it with GRASS GIS 9